sortinghat/core/utils.py

```diff
@@ -119,3 +120,32 @@ def unaccent_string(unistr):
     string = ''.join(cs)
 
     return string
+
+
+def validate_field(name, value, allow_none=False):
+    """Validate a given string field following a set of rules
+
+    The conditions to validate `value` consists on checking if its value is `None`
+    and if this is allowed or not; if its value is not an empty string or if it is
+    not composed only by whitespaces and/or tabs.
+
+    :param name: Name of the field to validate
```

Comment:
We start these descriptions with lowercase letters.

Reply:
Done, sorry about that.